lab_1_classify_by_unigrams/main.py

```diff
@@ -49,6 +85,31 @@ def compare_profiles(
     :param profile_to_compare: a dictionary of a profile to compare the unknown profile to
     :return: the distance between the profiles
     """
+    if not isinstance(unknown_profile, dict) or not isinstance(profile_to_compare, dict) \
+            or ('name' or 'freq') not in unknown_profile \
+            or ('name' or 'freq') not in profile_to_compare:
+        return None
+
+    unknown_freq = unknown_profile['freq']
+    compare_freq = profile_to_compare['freq']
+    mutual_characters = [[], []]
+    if isinstance(unknown_freq, dict) and \
```

Comment:
reverse the condition and return None as soon as posible

Reply:
Reversed